### DISCUSSION TITLE
first-line max width

### DISCUSSION THREAD

**Discussion by ercanertan** [tailwindlabs/tailwindcss#9112] [Ideas]
Hi,

I believe the use case is quite important but people using different solution for this.

I would like to control the width of first line of the content. 

Imagine I have a bit long title but this is coming from text field (**not** textarea or wysiwyg ).

Would be great if **first-line** ( Pseudo-elements ) would support max width. 

This is what I was trying to achieve.

first-line:max-w-[200px]

<img width="972" alt="Screenshot 2022-08-16 at 15 48 22" src="https://user-images.githubusercontent.com/13382249/184909859-8cde78ba-734c-4ec2-98cc-61a733f9b971.png">

I am not sure even possible but hope that make sense!

Thanks

**Comment by wongjn** [comment]
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/::first-line), `::first-line` does not allow the `max-width` CSS property. So this isn't a case that Tailwind does not support it, but rather that the underlying CSS specification not allowing the property.
